plugin.audio.soundcloud/resources/lib/kodi/cache.py

```diff
@@ -0,0 +1,31 @@
+import os
+import time
+
+
+class Cache:
```

Comment:
If you can point me to a good and maintained Kodi caching module, I'll use it instead of this module.